sku.0/sys.server/compiled/game/datatables/buildout/tatooine/tatooine_2_7.tab

```diff
@@ -1,3 +1,7 @@
-server_template_crc	cell_index	px	py	pz	qw	qx	qy	qz	scripts	objvars
-h	i	f	f	f	f	f	f	f	s	p
-object/building/kashyyyk/thm_kash_rodian_bannerpole_s01.iff	0	0	195	0	0	0	0	0		$|
+server_template_crc	cell_index	px	py	pz	qw	qx	qy	qz	scripts	objvars
+h	i	f	f	f	f	f	f	f	s	p
```

Comment:
when modifying what was a placeholder buildout you need to convert them such that they include all required fields, including the objid, which can start with -1 and decrement for each row, and the container.

Reply:
This actually works Ive been arguing for years but nobody believes me. Run it on a test vm.

Reply:
It works because there is backward compatibility for Buildout Version 1 in [`ServerBuildoutManager.cpp#L699`](https://github.com/SWG-Source/src/blob/master/engine/server/library/serverGame/src/shared/core/ServerBuildoutManager.cpp#L699) but that doesn't excuse it as an appropriate implementation for changes or additions.